frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/SelectSubformSection/SelectSubformSection.module.css

```diff
@@ -1,3 +1,14 @@
+.selectLayoutSet {
+  display: grid;
+  flex-direction: column;
+  gap: var(--fds-spacing-2);
+}
+
```

Comment:
This class does not seem to be in use
```suggestion
```

Reply:
Nice catch, it must have been mistakenly kept during the refactoring.